charts/redpanda/ci/04-one-node-cluster-no-tls-sasl-values.yaml

```diff
@@ -19,9 +19,8 @@ tls:
 auth:
   sasl:
     enabled: true
-    users:
-      - name: admin
-        password: hunter2
+    secretRef: "redpanda-users"
+
```

Comment:
Let's leave this one as name/password. Test 05 will test with secretRef and this way we have both cases covered.

Reply:
that is my fault, it was me testing defaults and i forgot to revert this, good catch!